pychromecast/discovery.py

```diff
@@ -116,26 +133,32 @@ def remove_service(self, _zconf, typ, name):
                     break
 
         if not cast_info:
-            _LOGGER.debug("remove_service unknown %s, %s", typ, name)
+            _LOGGER.debug("remove_service unknown %s, %s", type_, name)
             return
 
         if device_removed:
             self._cast_listener.remove_cast(uuid, name, cast_info)
         else:
             self._cast_listener.update_cast(uuid, name)
 
-    def update_service(self, zconf, typ, name):
+    def update_service(self, zc: zeroconf.Zeroconf, type_: str, name: str) -> None:
```

Comment:
Changing parameter names is a breaking change if the function is called with keyword arguments.

Reply:
This class implements the `zeroconf.ServiceListener` interface: https://github.com/python-zeroconf/python-zeroconf/blob/dfc9b8d7dec519ca713a811613122718cb2d733e/src/zeroconf/_services/__init__.py#L37-L45, the parameter names were previously not aligned with the interface.

Reply:
Ok.